database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Down/74_DML_PIMS_ROLE_CLAIM.sql

```diff
@@ -0,0 +1,44 @@
+BEGIN TRANSACTION;
+
+DECLARE @functional BIGINT;
+DECLARE @restricted BIGINT;
+DECLARE @readOnly BIGINT;
+DECLARE @systemAdministrator BIGINT;
+SELECT @functional = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Functional';
+SELECT @restricted = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Functional (Restricted)';
+SELECT @readOnly = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'Read Only';
+SELECT @systemAdministrator = ROLE_ID FROM PIMS_ROLE WHERE NAME = 'System Administrator';
+
+DECLARE @documentView BIGINT;
+DECLARE @documentAdd BIGINT;
+DECLARE @documentEdit BIGINT;
+DECLARE @documentDelete BIGINT;
+DECLARE @documentAdmin BIGINT;
+SELECT @documentView = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'document-view';
+SELECT @documentAdd = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'document-add';
+SELECT @documentEdit = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'document-edit';
+SELECT @documentDelete = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'document-delete';
+SELECT @documentAdmin = CLAIM_ID FROM PIMS_CLAIM WHERE NAME = 'document-admin';
+
+/* Administrator */
+DELETE FROM [dbo].[PIMS_ROLE_CLAIM] WHERE CLAIM_ID = @documentView AND ROLE_ID = @systemAdministrator;
```

Comment:
I guess we could simplify this by just deleting based on just the clam instead of the role? (since we are removing all instances of these claims from the db in this case.